### PR TITLE
Add contributing.md file

### PR DESCRIPTION
**Pull request by RitikJaiswal75**

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Detailed doc! 
Looks great to merge!

**Review by Maheima — APPROVED** [review]

**Review by harshith-venkatesh — APPROVED** [review]
Great explanation @RitikJaiswal75

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+This will register all the commands to your discord bot.
+
+Now let's link our local development server to our bot.
+
+- After all the commands are installed we need to save few secrets in wrangler cli.
+  - run command `wrangler secret put DISCORD_TOKEN` and then enter the value of your token.
+  - run command `wrangler secret put DISCORD_PUBLIC_KEY` and enter the public key.
+- Now, start the local server with the command `npm start`
+- Once the wrangler starts make sure it is running on port `8787`
+- Once the server starts on desired port open another terminal and type in the command `npm run ngrok`
```
Comment (prakashchoudhary07) [inline CONTRIBUTING.md:62]: Why on only 8787?

Comment (RitikJaiswal75) [reply CONTRIBUTING.md:62]: it's a default port provided by wrangler

**Pull request merged**